app/data_model/database.py

```diff
@@ -60,6 +60,19 @@ def __repr__(self):
         return "<EQSession('%s', '%s', '%s')>" % (self.eq_session_id, self.user_id, self.timestamp)
 
 
+class UsedToken(base):
+    __tablename__ = "used_token"
+    token = Column("token", String, primary_key=True)
```

Comment:
Maybe this should be called 'jwt_jti' to match the value that's being stored and for variable names in this class?